virttest/utils_misc.py

```diff
@@ -1562,6 +1562,49 @@ def add_identities_into_ssh_agent():
     process.run("ssh-add")
 
 
+def gen_lvmap_mpath(session):
```

Comment:
I think host also need this function also, so probably remove session parameter

Reply:
instead of removing, we can have session with default param as None, and handle it for host, when session is given let is work for guest.

Reply:
thank you both, I have updated code to both support host and session scenarios. Pls help review again~